app/main.py

```diff
@@ -1,3 +1,49 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int | float) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, number: int | float) -> Distance:
+        if isinstance(number, Distance):
+            return Distance(km=self.km + number.km)
+        else:
+            number = Distance(number)
+            return Distance(km=self.km + number.km)
```

Comment:
remove "km=" and redundant else. Also, why do you need this variable at line 18? You can just write `return Distance(self.km + number)` at line 19. https://github.com/mate-academy/py-car-wash-station/blob/master/checklist.md can help (search "Clean Code").